client-java/sql/src/main/java/org/evomaster/client/java/sql/QueryResultSet.java

```diff
@@ -0,0 +1,58 @@
+package org.evomaster.client.java.sql;
+
+import java.util.TreeMap;
+
+public class QueryResultSet {
```

Comment:
add JavaDoc, specifying what this class is for

Reply:
done